Tests/SentryTests/SentryHubTests.swift

```diff
@@ -10,7 +10,7 @@ class SentryHubTests: XCTestCase {
         let options: Options
         let error = NSError(domain: "", code: 0, userInfo: [NSLocalizedDescriptionKey: "Object does not exist"])
         let exception = NSException(name: NSExceptionName("My Custom exeption"), reason: "User wants to crash", userInfo: nil)
-        var client: TestClient!
+        lazy var client = TestClient(options: options)
```

Comment:
This is something I plan to do more of. It's nicer to have all the properties init'd right away, to avoid having distance between the declaration and initialization, so it's easier to read through what all the fixture has (and just more compact overall). The lazy vars help with properties using other properties. I also make use of `lazy var someVar: SomeType = {...}()` as well for properties that need multiple lines to be fully built.

Reply:
Great point. I will use `lazy` more in tests from now on.